### ISSUE TITLE
Missing AccessibilityNodeInfo.AddAction overload

### ISSUE THREAD

**Issue by GalaxiaGuy** [xamarin/xamarin-android#5368]
`Android.View.Accessibility.AccessibilityInfo`

is missing the overload:

`public void addAction (AccessibilityNodeInfo.AccessibilityAction action)`

https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#addAction(android.view.accessibility.AccessibilityNodeInfo.AccessibilityAction)

A deprecated overload is still available:

`public void addAction (int action)`

https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#addAction(int)
https://docs.microsoft.com/en-us/dotnet/api/android.views.accessibility.accessibilitynodeinfo.addaction?view=xamarin-android-sdk-9#Android_Views_Accessibility_AccessibilityNodeInfo_AddAction_Android_Views_Accessibility_Action_

But that does not provide the same functionality.

There is a comment here about it being intentional to work around a binding issue:
https://github.com/xamarin/xamarin-android/blob/8c86edabcad85fa8da9da93d0693a753c98d0e3f/src/Mono.Android/metadata#L1076
